debsources/app/copyright/routes.py

```diff
@@ -35,6 +35,23 @@ def skeleton_variables():
     lambda e: (ErrorHandler()(e, http=404), 404))
 
 
+# Before request
+@bp_copyright.before_request
+def before_request():
+    endpoint_w_versions = ['license', 'file']
```

Comment:
This code is more or less copied 3 times in each Blueprint. Any way to make this generic? (through an helper function for instance)